### PR TITLE
Maverick rnr link

### PR DESCRIPTION
**Pull request by robinheinze**
## Please verify the following:

- [x] `yarn ci:test` **jest** tests pass with new tests, if relevant

## Describe your PR

- Adds link to audio file when use presses on each episode card
- Adds Switch to filter by favorites (NOTE: @yulolimum is working on updating the design of this component in #2044, so it will look better after that)

### Visuals

https://user-images.githubusercontent.com/6894653/186535274-04273c19-3878-4953-8676-67a011e0ef76.mp4

https://user-images.githubusercontent.com/6894653/186535298-ab5a52a2-b51d-4f2e-8e95-bd3818d498d7.mp4

Screen Shot 2022-08-24 at 3 28 14 PM

### CODE REVIEW

**Comment by robinheinze** [comment]
Just realized there's a conflict between the touch events on the card and the like button. Working on a fix, stay tuned.

File: boilerplate/app/screens/DemoPodcastListScreen.tsx [hunk 1]
```diff
@@ -62,7 +78,11 @@ const EpisodeCard = observer(function EpisodeCard({
   isFavorite: boolean
 }) {
   return (
-    <View style={[$rowLayout, $item]}>
+    <TouchableOpacity
```
Comment (jamonholmgren) [inline boilerplate/app/screens/DemoPodcastListScreen.tsx:81]: Should we be using `Pressable` here?

File: boilerplate/app/utils/open-link-in-browser.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { Linking } from "react-native"
+
+export function openLinkInBrowser(url: string) {
```
Comment (jamonholmgren) [inline boilerplate/app/utils/open-link-in-browser.ts:3]:
```suggestion
/**
 * Helper for opening a give URL in an external browser.
 */
export function openLinkInBrowser(url: string) {
```

**Review by jamonholmgren — APPROVED** [review]
This looks great! Kinda fun too. Looking forward to trying it out.

**Pull request merged**